oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -47,9 +48,21 @@ async fn test_server() {
             .await
     });
 
+    let key_0 = Location {
+        latitude_millidegrees: 52_000,
+        longitude_millidegrees: 0,
+    }
+    .to_bytes();
+    let key_1 = Location {
+        latitude_millidegrees: 14_000,
+        longitude_millidegrees: -12_000,
+    }
+    .to_bytes();
+
     mock_static_server.set_response_body(test_utils::serialize_entries(hashmap! {
-        b"52,0".to_vec() => br#"{"temperature_degrees_celsius":10}"#.to_vec(),
-        b"14,12".to_vec() => br#"{"temperature_degrees_celsius":42}"#.to_vec(),
+        b"index".to_vec() => [key_0.clone(), key_1.clone()].concat(),
```

Comment:
What do you think about using `LookupData::for_test` instead (in a separate PR)? Or are these tests intended to actually check the functionality of `refresh`?

Reply:
Currently the tests do exercise at least the initial load via HTTP, which is nice, even though we don't test subsequent refreshes, so I think it's good to keep this logic around.